djstripe/models.py

```diff
@@ -704,25 +733,28 @@ def _attach_objects_hook(self, cls, data):
 
 @class_doc_inherit
 class Plan(StripePlan):
-    __doc__ = getattr(StripePlan, "__doc__")
+    """A model extending StripePlan providing dj-stripe functionality."""
+
+    __doc__ += getattr(StripePlan, "__doc__")
 
     # account = models.ForeignKey("Account", related_name="plans")
 
     class Meta(object):
+        """Meta class options."""
```

Comment:
I'm not sure metaclass comments are useful. Is there a specific rule we can ignore for metaclasses and __init__ methods?